Jellyfin.Api/Models/UserDtos/ForgotPasswordDto.cs

```diff
@@ -11,5 +11,5 @@ public class ForgotPasswordDto
     /// Gets or sets the entered username to have its password reset.
     /// </summary>
     [Required]
-    public string? EnteredUsername { get; set; }
+    required public string EnteredUsername { get; set; }
```

Comment:
Pretty sure the order is supposed to be `public required string` but analyzer has a bug
https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3527

Reply:
Thx I didn't know this, for now I'd just merge it as is (we already have code with this ordering) and fix it when the analyzer gets fixed.

Reply:
I agree, just wanted to raise awareness